packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2484,6 +2484,16 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   @override
   void userUpdateTextEditingValue(TextEditingValue value, SelectionChangedCause? cause) {
+    // TODO(chunhtai): Figure out why context or renderEditable can be null
+    // https://github.com/flutter/flutter/issues/79818.
+    if (_editableKey.currentContext == null || _editableKey.currentContext!.findRenderObject() == null) {
+      assert(
+        false,
+        'current context or render editable should not be null when '
+        'updating the text editing value'
```

Comment:
Nit: Capital at the start and period at the end?  Or maybe it doesn't matter in an error message.

Reply:
Is there any variable or anything we want to log here to help us figure this out when it happens?